semgrep/tests/e2e/snapshots/test_ci/test_full_run/autofix-azure-pipelines/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+    "meta": {
+        "semgrep_version": "<sanitized version>",
+        "repository": "/project_name/project_name",
```

Comment:
the leading slash is probably not supposed to be here 🤔

Reply:
Yeee this is the off by one error that I fixed (this comes from parsing the repo name from the github url)